dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipOutboundConnLink.java

```diff
@@ -48,7 +54,8 @@ public abstract class SipOutboundConnLink extends SipConnLink {
 	/** name of outbound chain */
 	private final String m_chainName;
 
-	private Bootstrap bootstrap = new Bootstrap();
+	private BootstrapExtended bootstrap;
+	private NettyFramework nettyFw;
 	private EventLoopGroup workerGroup = new NioEventLoopGroup();
```

Comment:
Should be able to remove this EventLoopGroup now that we're using the Netty bundle.

Reply:
If you could update the copyright too that'd be great.